src/Windows/R/Components/Impl/Resources.resx

```diff
@@ -442,12 +442,18 @@ If you want to delete all data on the remote machine you must delete your user p
   <data name="ConnectionManager_RemoveConnectionConfirmation" xml:space="preserve">
     <value>Are you sure you want to remove the '{0}' connection?</value>
   </data>
+  <data name="ContainerManager_ContainerServiceIsNotInstalled" xml:space="preserve">
+    <value>Docker service isn't installed.\nTo start working with containers, install Docker for Windows and restart VS.</value>
```

Comment:
We usually auto-open the download page. `ProcessServices.Start('https://www.docker.com/docker-windows')`

Reply:
Do we want to open it every time user clicks on the "Containers" button?

Reply:
Only after reporting the error. Same as in say, markdown publishing and pandoc

Reply:
Error is reported in the watermark. I can add a clickable link to the message. Will that work?

Reply:
Sure